pkg/database/sqlite/config.go

```diff
@@ -1,17 +1,44 @@
 package sqlite
 
+import (
+	"os"
+	"path/filepath"
+)
+
 // Config implements database.Configurator interface and
 // handles the default configuration parameters of the sqlite database.
 type Config struct {
 	Path string
+	Home string
 }
 
 func (c *Config) SetDefaults() {
-	if c.Path == "" {
-		c.Path = "storage.db"
+	if c.Path != "" {
+		return
 	}
+
+	c.Path = DefaultPath(c.Home)
 }
 
 func (c *Config) Validate() error {
 	return nil
 }
+
+// DefaultPath returns the default sqlite path for the given home directory.
+func DefaultPath(home string) string {
+	if home == "" {
+		home = os.Getenv("TERRALIST_HOME")
+	}
+
+	if home == "" {
+		if userHome, err := os.UserHomeDir(); err == nil {
+			home = userHome
+		}
+	}
```

Comment:
This check is also redundat. By default (if the home flag is not provided), Terralist will default to `~/.terralist.d` as of https://github.com/terralist/terralist/blob/master/cmd/terralist/server/flags.go#L129 (the PathFlag implementation automatically expands `~` to the user's home dir as of https://github.com/terralist/terralist/blob/master/pkg/cli/path_flag.go#L87-L107).